### DISCUSSION TITLE
schemas ??

### DISCUSSION THREAD

**Discussion by FVilli** [juicyllama/llana#159] [Q&A]
Hi guys ! What about db schemas ? What's happens if i have 2 table with same name in two different schemas ?

**Comment by andyslack** [comment]
Hi @FVilli 

If you have two data sources (schemas), then you would need to run two llana instances, you can then have some logic to pull from the correct llana endpoint. 

I hope this makes sense, if I had more context I could advise better.

**Reply by FVilli** [reply]
Yes, if I can move the schemas to a different database. If I had a database with multiple schemas, we would need something to choose or skip schemas. Anyway I cloned the repository and was working on it... but only for postgres...

**Reply by andyslack** [reply]
Do let me know how you get on and if you have any further questions.
